docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,594 @@
+{
```

Comment:
<p>Just a really small comment as this may come up in the future.</p><p> </p><p>I understand what you are trying to accomplish here but the choice of <code>cand[0]</code>  seems arbitrary and is largely dependent on the window size. It is possible that the candidates are scattered across the time series. So, it may actually be better to do:</p><p> </p><pre class="ql-syntax" spellcheck="false">plt.plot(ts, c='b')
plt.title(f'The most left candidate (starting at: {idx})')
for idx in cand:
    plt.plot(np.arange(idx, idx+m), ts[idx:idx+m], c='r')
plt.show()
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
Yes. That is correct. Although I explained this after the figure, the choice of `cand[0]` can still bother the reader. I will do what you suggested (similar to the plot of first figure). Will be fixed in the next commits.